ruby/README.md

```diff
@@ -827,6 +824,179 @@ class TestClass
 end
 ```
 
+### SOLID Object Oriented Design
+These concepts should be applied generally when writing Object Oriented Code. Read more [here](https://en.wikipedia.org/wiki/SOLID).
+
+#### Single Responsibility Principle
+> "Gather together the things that change for the same reasons. Separate those things that change for different reasons."
+> -- Robert C. Martin
+
+Design your classes such that all public methods relate to a singular purpose.
+Similarly, define your methods such that each only accomplishes a single task.
+
+> TIP: Describe your class aloud and listen for usage of the word: "AND"
```

Comment:
⛏️❓ This is quoted in Markdown but is not styled as if it were a quote and does not give a source person. Is this intentional?

🔧 Assuming this is not a quote, I'd suggest appending an explanation of **why** to look for the word "AND"

E.g. 
```
Describe your class aloud and listen for usage of the word: "AND". Usage of "and" often implies the class has multiple responsibilities and can be refactored to have fewer responsibilities."
```